core/themes/basis/css/component/comment.css

```diff
@@ -160,6 +160,10 @@
   padding: 0 0.5em;
 }
 
```

Comment:
Since this new ruleset and the previous one are related, we don't need a blank line between them.